AnalysisNtuple/makeAnalysisNtuple.C

```diff
@@ -718,7 +798,10 @@ makeAnalysisNtuple::makeAnalysisNtuple(int ac, char** av)
 
 
 	    InitVariables();
-	    FillEvent(year);
+	   // FillEvent(year);
+
+	   if(applyHemVeto) FillEvent(year,isHemVetoObj); //HEM test
+	   else FillEvent(year,false);
```

Comment:
Instead of calling FillEvent differently, add the applyHemVeto in as a condition for isHemVetoObj in line 781, then when applyHemVeto is false, isHemVetoObj would also be false